mindsworth/My-Diary-Frontend
# #162345973 Setup empty react project

#### What does this PR do?
React setup

#### Description of Task to be completed?
- Installed all necessary package
- Configured Webpack
- Configured babel

#### How should this be manually tested?
1. After cloning or pulling the branch to your local
2. Run yarn install to install all necessary packages and run yarn start:dev start your app on port 8080
3. visit the right URL e.g http://localhost:8080 to see the React Setup.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
-[ #162345973](https://www.pivotaltracker.com/story/show/162345973)

#### Screenshots (if appropriate)
<img width="1677" alt="screenshot 2018-11-30 at 10 24 39 pm" src="https://user-images.githubusercontent.com/14542449/49315672-cd62d400-f4ee-11e8-84d7-fab62dbd6ed7.png">

#### Questions:
N/A
